apis/v1alpha1/snippetsfilter_types.go

```diff
@@ -47,12 +51,14 @@ type Snippet struct {
 	Context NginxContext `json:"context"`
 
 	// Value is the NGINX configuration snippet.
+	// +kubebuilder:validation:MinLength=1
 	Value string `json:"value"`
 }
 
 // NginxContext represents the NGINX configuration context.
 //
 // +kubebuilder:validation:Enum=main;http;http.server;http.server.location
+// +kubebuilder:validation:MinLength=4
```

Comment:
Our NGF code already covers the Enum validation, though I don't see it covering the MinLength, do you think thats just unnecessary?

Reply:
We will catch a MinLength violation in the switch statement on context (it will hit the default case), so we don't need to check the length explicitly. 

I did some more testing with the CRD, and we can remove MinLength. `Context` will still be a required field.